docs/UserInterface.md

```diff
@@ -3,6 +3,8 @@
 Welcome to the Taxonomy Development Tools User Interface Guide. This document is designed to provide comprehensive details on navigating and utilizing the TDT interface efficiently. Whether you are looking to manage data tables, edit information, or leverage advanced features, this guide will assist you in making the most out of TDT.
 
 1. [Tables](#tables)
+   1. [Switch system tables](#switch-system-tables)
```

Comment:
Provisional notes:
 - Hoping that these are mostly hidden in the new release.  Let's review when we can see it.
 - Switch system tables is a bit of a confusing name